src/base/detoken.lua

```diff
@@ -56,16 +57,26 @@
 			setfenv(func, e)
 
 			-- run it and get the result
-			local result = func() or ""
+			local success, result = pcall(func)
+			if not success then
+				err    = result
+				result = nil
+			else
+				err    = nil
```

Comment:
Wouldn't this already be `nil` since this function returns on `loadstring` failure?